src/components/appointments/CreateAppointment/CreateAppointment.js

```diff
@@ -66,11 +66,15 @@ const CreateAppointment = () => {
               {isLoading ? (
                 <Loader modifiers={['centered']} />
               ) : (
-                <RequireSession expired={sessionExpired} expiryDate={expiryDate} onNavigate={reset}>
+                <SessionModalTracker
+                  expired={sessionExpired}
+                  expiryDate={expiryDate}
+                  onNavigate={reset}
+                >
```

Comment:
```suggestion
                <SessionModalTracker expiryDate={expiryDate}>
```
the `expired` and `onNavigate` props were obsoleted - you'll likely be able to do some more cleanups with unused references after this change :)